magefile.go

```diff
@@ -827,6 +839,43 @@ func packageAgent(requiredPackages []string, packagingFn func()) {
 	mg.Deps(CrossBuild, CrossBuildGoDaemon)
 	mg.SerialDeps(devtools.Package, TestPackages)
 }
+func copyComponentSpecs(componentName, versionedDropPath string) (string, error) {
+	sourceSpecFile := filepath.Join("specs", componentName+specSuffix)
+	err := devtools.Copy(sourceSpecFile, filepath.Join(versionedDropPath, componentName+specSuffix))
+	if err != nil {
+		return "", errors.Wrapf(err, "failed copying spec file '%s' to '%s'")
```

Comment:
Does this error string fill correctly? also `%q` can be used to quote a string

Reply:
of course it doesn't :-)